src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
Because RemoteExecutorHandle.Process is null in UAP. So `Process P` will be null and when we do p.WaitForExit() it will give a NullReferenceException. What we could do instead of creating it's own test is:

```cs
using (Process p = RemoteInvoke(s => int.Parse(s), expectedExitCode.ToString(), new RemoteInvokeOptions { ExpectedExitCode = expectedExitCode }).Process)
{
       if (PlatformDetection.IsWinRT)
      {
           // In UAP RemoteExecutor returns null as the process. When doing RemoteInvoke it already checked for the expected exit code.
          return;
      }       
     ...
}
```

Would you rather have that?